src/Package.php

```diff
@@ -168,6 +177,12 @@ public function hasMigration(string $migrationFileName): static
 
     public function hasMigrations(...$migrationFileNames): static
     {
+        if (empty($migrationFileNames)) {
```

Comment:
So for packages that don't have migrations explicitly specified, we'd auto discover them by default.

This will cause that for all packages that don't have migrations at all, we'll still check the `migrations` directory, which has an - albeit small - performance impact.

I'd be more comfortable merging this in, if people would only specifically opt-in to the checking behavior. What do you think?

Reply:
@freekmurze you think remove this check and just require package devs to explicitly say "just discover my migrations"